### DISCUSSION TITLE
commenting on artworks

### DISCUSSION THREAD

**Discussion by fuckyougithub1111** [3urobeat/steam-comment-service-bot#67] [Ideas]
well the title says it all. is it possible for you to make a feature to comment on steam artwork comments?

**Comment by 3urobeat** [comment]
Oh hey you are the first one to use the discussions tab :D

The libraries I am using to communicate with Steam don't support commenting on artworks as it seems. I would need to look into what happens when you would press on the comment button yourself and if I could imitate that somehow (using a post request?). That would probably take some time and isn't a priority right now.  
I am going to note it down but I'm probably not going to work on it anytime soon as I see this project feature-wise as completed and I have other projects that I want to complete right now.

**Comment by 3urobeat** [comment]
Sorry for the necrobump but I wanted to let you know that this feature has since been implemented in version 2.13.0. You can now comment and vote on sharedfiles (e.g. screenshots, artworks and guides).
